File: Java/Carrera.java
import java.util.ArrayList;
import java.util.List;

public class Carrera {
    private String nombre;
    private int duracion;
    
    public Carrera(String nombre, int duracion) {
        this.nombre = nombre;
        this.duracion = duracion;
    }
    
    public String getNombre() {
        return nombre;
    }
    
    public void setNombre(String nombre) {
        this.nombre = nombre;
    }
    
    public int getDuracion() {
        return duracion;
    }
    
    public void setDuracion(int duracion) {
        this.duracion = duracion;
    }
    
    public void imprimirDatos() {
        System.out.println("Nombre de la carrera: " + nombre);
        System.out.println("Duración de la carrera: " + duracion + " años");
    }
}

File: Phyton/Carrera.py
class Carrera:
    def __init__(self, nombre, duracion):
        self.nombre = nombre
        self.duracion = duracion
    
    def get_nombre(self):
        return self.nombre
    
    def set_nombre(self, nombre):
        self.nombre = nombre
    
    def get_duracion(self):
        return self.duracion
    
    def set_duracion(self, duracion):
        self.duracion = duracion
    
    def imprimir_datos(self):
        print("Nombre de la carrera: " + self.nombre)
        print("Duración de la carrera: " + str(self.duracion) + " años")